onboarding/views.py

```diff
@@ -1,39 +1,84 @@
 from django.shortcuts import render
 from django.http import HttpResponse
 
-from openpyxl import load_workbook
 from course.models import Course, CourseCategory
 
-import_sheets = {
+from rest_framework.views import APIView
+from .serializers import UploadSerializer
+# from rest_framework import MultiPartParser
+from .onboarding import Upload
+
+from openpyxl import load_workbook
+from django.core.files.storage import default_storage
+from django.core.files.base import ContentFile
+
+from .process_files import process_import
+
+class FileUpload(APIView):
+    # parser_classes = [MultiPartParser, FormParser]
+
+    def post(self, request, format=None):
+        file_upload = request.FILES['file_upload']
+        file_name = file_upload.name
+        file_type = request.data['file_type']
+        if file_name != file_choices[file_type]:
+            return HttpResponse("wrong file type")
+
+        path = default_storage.save('tmp/curr.xlsx', ContentFile(file_upload.read()))
+        wb = load_workbook(filename=path)
+        for row in wb.iter_rows(min_row=1, max_row=1):
+            for i, cell in enumerate(row):
+                if cell != column_names[file_type][i]:
+                    return HttpResponse(
```

Comment:
Adding cell and column_names[file_type][i] to the error explanation would help users know what to fix

ex) Expected __ but found __ at ___